br/pkg/lightning/backend/local/local.go

```diff
@@ -1464,17 +1464,8 @@ func (local *Backend) ImportEngine(
 		log.FromContext(ctx).Info("engine contains no kv, skip import", zap.Stringer("engine", engineUUID))
 		return nil
 	}
-	kvRegionSplitSize, kvRegionSplitKeys, err := getRegionSplitSizeKeys(ctx, local.pdCtl.GetPDClient(), local.tls)
-	if err == nil {
-		if kvRegionSplitSize > regionSplitSize {
-			regionSplitSize = kvRegionSplitSize
-		}
-		if kvRegionSplitKeys > regionSplitKeys {
-			regionSplitKeys = kvRegionSplitKeys
-		}
-	} else {
-		log.FromContext(ctx).Warn("fail to get region split keys and size", zap.Error(err))
-	}
+
+	regionSplitSize, regionSplitKeys, _ = local.GetRegionSplitSizeKeys(ctx, regionSplitSize, regionSplitKeys)
```

Comment:
in the old code, `regionSplitSize` and `regionSplitKeys` is unchanged when error happens. But now they will become 0?

And `regionSplitSize` should become the larger value of {task configuation, cluster configuration}. Now it's the smaller value.